spec/system/add_orcid_spec.rb

```diff
@@ -12,31 +12,12 @@
     it "takes a user to their homepage after login", js: true do
       visit user_path(user)
       expect(page).to have_content "Welcome"
-      expect(page).to have_content "You do not have an ORCiD on file."
-      click_on "Edit"
+      expect(page).to have_content "You do not have an ORCID iD on file."
+      click_on "Add one"
       expect(page).to have_content "Editing User"
```

Comment:
Notice that now we only nag the user to enter their ORCID if there is none. If they have an ORCID already on file nothing is displayed.